packages/space-opera/src/app.ts

```diff
@@ -21,7 +21,12 @@
 
 import '@material/mwc-button';
 import '@polymer/paper-dialog/paper-dialog';
-
+// The order of imports is important, as this is the order in which their
+// stateChanged() methods will be called. We put the preview first to apply
+// changes to the <model-viewer> element in case they need to be queried by
+// subsequent components. Keep in mind that those will still have to await
+// updateComplete since rendering is async in Lit Element.
+import './components/model_viewer_preview/model_viewer_preview.js';
```

Comment:
Does the model_viewer_preview actually expose API to make changes to the model? Just trying to understand why it's called preview but its applying changes.

Reply:
The preview is where the model-viewer instance lives, so yes, it effectively exposes the whole MV API to the editor features.

Reply:
Other components only use getters on the MV  element. The preview  is the one place where the config Redux state is applied to the MV element, which happens when its  render method is called.